### PR TITLE
Review: README, Gemfile, Rakefile

### PR DESCRIPTION
**Pull request by moonglum**
# README
- Unified Veritas to begin with an upper-case letter.
- Changed the Copyright to include 2012.
- Added a full-stop to the end of the first sentence.
- Fixed two typos.
- Two small, stylistic changes.
# Gemfile and Rakefile

I made some changes to the Gemfile and Rakefiles. Maybe a little bit to much for a first pull request :( I separated everything into small commits, so you can review each one individually:
- Pelusa on Rubygems is the current version, therefore no require from github necessary
- Removed unused Dependencies
- Flay added for all Ruby Versions, not just 1.8
- Instead of MetricFu, run the metrics individually (very opinionated commit)
- Removed an unnecessary require from a rake task
- Jeweler and RCov should have their own Rakefiles (opinionated commit)
- Namespaced all metrics under `metrics`  (opinionated commit)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,28 +161,27 @@ The third phase of this project will be to add a few NoSQL engines (like [MongoD
...
+* If you want your code merged into the mainline, please discuss the proposed changes with me before doing any work on it. This library is still in early development, and the direction it is going may not always be clear. Some features may not be appropriate yet, may need to be deferred until later when the foundation for them is laid, or may be more applicable in a plugin.
 * Fork the project.
 * Make your feature addition or bug fix.
   * Follow this [style guide](https://github.com/dkubb/styleguide).
 * Add specs for it. This is important so I don't break it in a future version unintentionally. Tests must cover all branches within the code, and code must be fully covered.
-* Commit, do not mess with Rakefile, version, or history.  (if you want to have your own version, that is fine but bump version in a commit by itself I can ignore when I pull)
```
Comment (moonglum) [inline README.md:181]: I thought about putting the full stop after the closing brackets (my personal preference ;) ) – but different style guides disagree on that and I don't know which one you follow ;)

**Comment by dkubb** [comment]
@moonglum I added a few comments to this PR. Overall it's pretty good. I think there's only really one thing to change, and one thing that needs to be checked.

It's also great that you're reviewing the gem deps and rake tasks. I'd added them a while ago, and haven't reviewed them with a fresh perspective. After seeing your changes they make sense and will make things easier for people new to the project to understand.

**Comment by moonglum** [comment]
I'll go to bed now (it's nighttime here – and I have to get up early tomorrow ;) ) – I hope to get the rest done tomorrow :) Thank you for all your feedback!

**Comment by moonglum** [comment]
@dkubb Now your feedback is reflected in the commit :)

**Comment by dkubb** [comment]
@moonglum I'm going to merge this in as-is, and then revert that commit that removes the jruby group.

Thanks for this pull request!

**Pull request merged**